src/nextjs/server/index.tsx

```diff
@@ -308,7 +308,12 @@ async function isAuthenticated(token: string | null): Promise<boolean> {
       {},
       { token: token },
     );
-  } catch {
-    return false;
+  } catch (e: any) {
+    if (e.message.includes("Could not find public function")) {
+      throw new Error(
+        "Server Error: could not find api.auth.isAuthenticated. convex-auth 0.0.76 introduced a new export in convex/auth.ts. Add `isAuthenticated` to the list of functions returned from convexAuth(). See convex-auth changelog for more https://github.com/get-convex/convex-auth/blob/main/CHANGELOG.md",
+      );
+    }
+    throw e;
```

Comment:
I think you need to return false here. Otherwise errors like sending an invalid JWT to the Convex backend will also throw (which is why that one test is failing, I think).

Reply:
Ah for the token expired and invalid token cases, got it. Maybe we can pick those out too, for now I'll log whatever error it is.